### PR TITLE
deno: update to 1.5.2

### PR DESCRIPTION
**Pull request by eatradish**
Topic Description
-----------------

deno: update to 1.5.2

Package(s) Affected
-------------------

deno 1.5.2

Security Update?
----------------

No

Architectural Progress
----------------------

- [ ] AMD64 `amd64`   
- [ ] AArch64 `arm64`
    

Post-Merge Architectural Progress
---------------------------------

- [ ] AMD64 `amd64`   
- [ ] AArch64 `arm64`

### CODE REVIEW

File: extra-js/deno/autobuild/prepare [hunk 1]
```diff
@@ -1,4 +1,7 @@
 export V8_FROM_SOURCE=1
 export CLANG_BASE_PATH='/usr/'
-export GN_ARGS="no_inline_line_tables=false use_lld=false"
-
+export GN_ARGS="no_inline_line_tables=false use_lld=false is_clang=false \
```
Comment (MingcongBai) [inline extra-js/deno/autobuild/prepare:3]: `abinfo` please.

**Review by MingcongBai — CHANGES_REQUESTED** [review]

**Comment by eatradish** [comment]
photo_2020-11-09_14-57-27

emmm

**Comment by MingcongBai** [comment]
@liushuyu What do you say about this?

**Comment by liushuyu** [comment]
Please update to `deno` 1.5.3 and remove the patches.

**Comment by MingcongBai** [comment]
@eatradish Please update. Closing.

**Pull request closed without merging**